README.md

```diff
@@ -375,13 +375,36 @@ Edit this line and replace `127.0.1.1` with your public IP.
 
 ## Example Playbook
 
-This is an example, of how to use this role. Warning: the value of the Variables should be changed!
+This is an example of how to use this role. *Warning:* the values of the variables should be changed!
 
-```yaml
-    - hosts: servers
-      roles:
-         - { role: ebbba.bigbluebutton, bbb_coturn_secret: ee8d093109a9b273, bbb_greenlight_secret: 107308d54ff4a5f, bbb_greenlight_db_password: 2585c27c785e8895ec, bbb_letsencrypt_email: mail@example.com }
+Assuming the following directory structure:
+```
+├── ansible
+    ├── roles
+    │   └── ansible-role-bigbluebutton
+    ├── playbooks
+    │   └── bigbluebutton.yml
+    └── inventory
+	├── hosts
+        └── host_vars
+            └── your-domain.example.com
+                └── vars.yml
 ```
+You can clone the repository in your roles directory and then follow these steps inside the parent directory:
+
+1. Copy the sample inventory hosts file or append its containt to your already existing hosts file(`cp roles/ansible-role-bigbluebutton/examples/hosts inventory/hosts`)
+
+1. Edit the inventory hosts file (`inventory/hosts`) to include all the hosts you want in the `bigbluebutton` group.
+
+1. Create a directory for each of your servers to hold its configuration (`mkdir inventory/host_vars/<your-domain>`)
+
+1. Copy the sample configuration file to each of the servers configuration directory (`cp roles/ansible-role-bigbluebutton/examples/vars.yml inventory/host_vars/<your-domain>/vars.yml`)
+
+1. Edit the configuration file (`inventory/host_vars/<your-domain>/vars.yml`) to your liking. Event though all the variables are explained above, you may also take a look at `roles/ansible-role-bigbluebutton/defaults/main.yml` and see if there's something you'd like to copy over and override in your `vars.yml` configuration file.
+
+1. Copy the sample playbook (`cp roles/ansible-role-bigbluebutton/examples/playbook/bigbluebutton.yml playbooks`)
+
+1. Run the playbook using `ansible-playbook -i inventory/hosts playbooks/bigbluebutton.yml`.
```

Comment:
The numbering should go on, shouldn't it?

Reply:
This is the markdown way of generating a numbered list. If you see the way the Readme is previewed, it is displayed correctly. Maybe it is still better to change the numbering though, for those that view the file locally.

Reply:
Fixed